src/components/layout/content-block-container.tsx

```diff
@@ -8,3 +9,11 @@ export const ContentBlockContainer = styled.div`
     margin-top: 180px;
   }
 `;
+
+export const ContentBlockContainerWithoutHero = styled.div`
```

Comment:
Funny name 😁 This is ok to share from here for now.
Once we have all pages I would suggest to work on the overall layout again to see what we are missing. 

Because a simple page like imprint with no header. That should not require any additional container. The Layotu should handle this. 

Your change is good, just some loud thought👍